packages/graphql/schemas/schema.graphql

```diff
@@ -1576,6 +1581,15 @@ type Query {
   wizard: Wizard!
 }
 
+"""Properties describing a React component"""
+type ReactComponentDescriptor {
+  """A description for the component"""
+  description: String!
```

Comment:
Just to revisit - I don't think we ever consume `description` in the UI - this seems like an unused field, and doesn't seem to contain anything useful in practice.